### ISSUE TITLE
BleachBit 3.0 Not Clearing Google Chrome History

### ISSUE THREAD

**Issue by sibernetik** [bleachbit/bleachbit#744]
I saw that there were similar posts about this, but all of them were on older versions of BleachBit and appear to have been resolved. I am running BleachBit 3.0 on a Windows 10 (1809) device with Google Chrome 79.0.3945.117 (64-bit). When I run BleachBit I get the following error:

```
Error: google_chrome.history: Function: Clean file: C:\Users\<redacted>\AppData\Local\Google\Chrome\User Data\Default\History
Traceback (most recent call last):
  File "bleachbit\Worker.pyo", line 88, in execute
  File "bleachbit\Command.pyo", line 162, in execute
  File "bleachbit\Special.pyo", line 199, in delete_chrome_history
  File "bleachbit\FileUtilities.pyo", line 406, in execute_sqlite3
OperationalError: no such column: lower_term: C:\Users\<redacted>\AppData\Local\Google\Chrome\User Data\Default\History
ERROR:bleachbit.Worker:Error: google_chrome.history: Function: Clean file: C:\Users\<redacted>\AppData\Local\Google\Chrome\User Data\Default\History
Traceback (most recent call last):
  File "bleachbit\Worker.pyo", line 88, in execute
  File "bleachbit\Command.pyo", line 162, in execute
  File "bleachbit\Special.pyo", line 199, in delete_chrome_history
  File "bleachbit\FileUtilities.pyo", line 406, in execute_sqlite3
OperationalError: no such column: lower_term: C:\Users\<redacted>\AppData\Local\Google\Chrome\User Data\Default\History
```

Any help with this would be greatly appreciated.

**Comment by sibernetik** [comment]
Update: I did some more digging and I found #565. I went ahead and removed the .ldb file from C:\Users(my username)\AppData\Local\Google\Chrome\User Data\Default\Session Storage and, while I no longer get the original error, it still does not appear to be clearing the Chrome history. Also, I now get a new error:

```
Error: google_chrome.history: Function: Clean file: C:\Users\<redacted>\AppData\Local\Google\Chrome\User Data\Default\Favicons
Traceback (most recent call last):
  File "bleachbit\Worker.pyo", line 88, in execute
  File "bleachbit\Command.pyo", line 162, in execute
  File "bleachbit\Special.pyo", line 119, in delete_chrome_favicons
  File "bleachbit\Special.pyo", line 37, in __get_chrome_history
  File "bleachbit\Special.pyo", line 81, in get_sqlite_int
OperationalError: no such table: meta
```

**Comment by az0** [comment]
@sibernetik 

I added two fixes. In a new minutes, there will be a binary available on the [CI site](https://ci.bleachbit.org/). Please test build version 3.1.0.1477.

**Comment by sibernetik** [comment]
I just installed 3.1.0.1477 and that appears to have done the trick! I no longer see errors related to Google Chrome and the history looks to have been cleared successfully.

**Comment by az0** [comment]
@sibernetik 
Thanks for confirming. This bug report came just in time for the BleachBit 3.1.0 beta release

**Comment by az0** [comment]
I published this fix in [BleachBit 3.1.0 beta](https://www.bleachbit.org/news/bleachbit-31-beta)
